Pipfile

```diff
@@ -74,6 +74,7 @@ ckanext-blob-storage = {editable = true, ref = "5b2f59217e8f88a3f2cdb9dc50bf4099
 ckanext-scheming = {editable = true, ref = "8d62a28370c4f231dbf2f9688ede93a0356410e3", git = "https://github.com/ckan/ckanext-scheming.git"}
 ckanext-versions = {editable = true, ref = "3a6094acd730f3495b0f8b99b14a2340042a721b", git = "https://github.com/datopian/ckanext-versions"}
 ckanext-dcat = {editable = true, ref = "51d651315ff2ff594ebd6eb1d658bc7e9553310c", git = "https://github.com/ckan/ckanext-dcat"}
+ckanext-short-urls = {editable = true, ref = "2205da8739b3e237a27521f08bfbb646b5cc5bc8", git = "https://github.com/fjelltopp/ckanext-short-urls"}
```

Comment:
I think we should start moving to tagged version numbers rather than commit hashes as our refs, e.g. `ref = "1.0.0"`